src/top/metrics.rs

```diff
@@ -49,47 +51,84 @@ async fn component_removed(client: Arc<SubscriptionClient>, tx: state::EventTx)
     }
 }
 
-async fn processed_events_totals(
-    client: Arc<SubscriptionClient>,
-    tx: state::EventTx,
-    interval: i64,
-) {
-    let res = client.component_processed_events_totals_subscription(interval);
+async fn events_in_total(client: Arc<SubscriptionClient>, tx: state::EventTx, interval: i64) {
```

Comment:
```suggestion
async fn events_in_totals(client: Arc<SubscriptionClient>, tx: state::EventTx, interval: i64) {
```

Nit: Similar functions use the plural form.